aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -275,12 +289,14 @@ module aptos_framework::account {
     /// 1. During normal key rotation via `rotate_authentication_key` or `rotate_authentication_key_call`
     /// 2. During resource account initialization so that no private key can control the resource account
     /// 3. During multisig_v2 account creation
-    public(friend) fun rotate_authentication_key_internal(account: &signer, new_auth_key: vector<u8>) acquires Account {
+    public(friend) fun rotate_authentication_key_internal(
+        account: &signer, new_auth_key: vector<u8>
+    ) acquires Account {
```

Comment:
This is kinda weird, it should either be one line, or all of the variables should be on different lines
```suggestion
    public(friend) fun rotate_authentication_key_internal(
        account: &signer,
        new_auth_key: vector<u8>
    ) acquires Account {
```

Reply:
But some users prefer the current style. Regarding the output you mentioned, movefmt aligns with your idea when it encounters more than four parameters or when the parameter string is very complex.